tests/erc721/erc721_test.go

```diff
@@ -543,6 +545,81 @@ func TestOpenSeaProxyPreApprovalOptInOut(t *testing.T) {
 	}
 }
 
+func TestRoyalties(t *testing.T) {
```

Comment:
Please also add a test for `onlyOwner`. See `revert.OnlyOwner.Diff()`

Reply:
already did so in 15bbd92df0d41c0e7fc7b65b6ea959f472dece18 (maybe I pushed it after you already started reviewing)